getting_started/workflow/export/exporting_for_android.rst

```diff
@@ -9,24 +9,33 @@ The following steps detail what is needed to setup the SDK and the engine.
 Download the Android SDK
 ------------------------
 
-Download and install the Android SDK from
-`developer.android.com <https://developer.android.com/studio/>`__.
+Download and install the Android SDK. 
 
-If you install Android Studio, you need to run it once to complete the SDK setup.
+- You can install it using `Android Studio <https://developer.android.com/studio/>`__.
+  
+  - Run it once to complete the SDK setup using these `instructions <https://developer.android.com/studio/intro/update#sdk-manager>`__.
+  - Ensure that the `required packages <https://developer.android.com/studio/intro/update#recommended>`__ are installed as well.
 
-Install OpenJDK 8
+- You can install it using the `command line tools <https://developer.android.com/studio/#command-tools>`__.
+  
+  - Once the command line tools are installed, run the `sdkmanager <https://developer.android.com/studio/command-line/sdkmanager>`__ command to complete the setup process:
+
+::
+
+    sdkmanager --sdk_root=<android_sdk_path> "platform-tools" "build-tools;30.0.1" "platforms;android-29" "cmdline-tools;latest"
```

Comment:
`"platform-tools"` doesn't seem valid:
```
$ ./sdkmanager --sdk_root=~/Project/godot/android/sdk "platform-tools" "build-tools;30.0.1" "platforms;android-29" "cmdline-tools;latest"
Warning: Failed to download any source lists!                                   
Warning: IO exception while downloading manifest                                
Warning: Still waiting for package manifests to be fetched remotely.            
Warning: Failed to find package 'platform-tools'
```

Reply:
Nevermind, it's the download failure which makes it not recognized. Likely another problem with OpenJDK 11 on my distro.